Tests/WorkspaceTests/ManifestSourceGenerationTests.swift

```diff
@@ -19,21 +19,6 @@ import TSCBasic
 import Workspace
 import XCTest
 
-extension String {
-    fileprivate func nativePathString(escaped: Bool) -> String {
-#if _runtime(_ObjC)
-        return self
```

Comment:
not sure if it matter but we lost this?

Reply:
Yeah ... I kinda gave up and started pulling in a bit more of the other changes to just centralise all the uses.  I like the `_` prefixed version better because it was really bothering me that we had an internal API visible.